docs/linting/TROUBLESHOOTING.md

```diff
@@ -241,3 +241,18 @@ However this involves a _lot_ of computations across every single token in a fil
 Across a large codebase, these can add up, and severely impact performance.
 
 We recommend not using this rule, and instead using a tool like [`prettier`](https://www.npmjs.com/package/prettier) to enforce a standardized formatting.
+
+## I am running into errors when parsing Typescript in my .vue files
```

Comment:
A previous entry on the page mentions Vue already: _I use a framework (like Vue) that requires custom file extensions, ..._. What do you think about moving this entry to just after that other one, to group Vue entries together?

Reasoning: if someone is reading about Vue, they probably want to see both FAQs.

Reply:
good idea!